site/src/components/Loader/Loader.tsx

```diff
@@ -18,7 +18,7 @@ export const Loader: FC<LoaderProps> = ({
 			data-testid="loader"
 			{...attrs}
 		>
-			<CircularProgress size={size} />
+			<Spinner aria-label="Loading..." size={size} />
```

Comment:
I might be misunderstanding but it seems we have a `Loader` component wrapping a `Spinner` component wrapping a `CircularProgress` component. Is there a reason to have 3 levels nesting here?

Reply:
Good point! The purpose of the 'Loader' is to act as a container that centrally positions the loading spinner on the page or within a section, with some padding around it. Meanwhile, the 'Spinner' can be utilized in other components, such as loading buttons. Perhaps 'Loader' is not the most suitable name, but I can't think of anything better at the moment.

Reply:
Ahh, makes sense! `Loader` works for me :)